ui/apps/platform/src/messages/standards.js

```diff
@@ -6,8 +6,6 @@ export const standardLabels = {
     [standardTypes.NIST_SP_800_53_Rev_4]: 'NIST SP 800-53',
     [standardTypes.HIPAA_164]: 'HIPAA 164',
     [standardTypes.CIS_Kubernetes_v1_5]: 'CIS Kubernetes v1.5',
-    [standardTypes.CIS_Docker_v1_1_0]: 'CIS Docker v1.1.0',
```

Comment:
Hmm, looks like this is an artifact of an earlier removal.

Reply:
Yup, this is from previous cis docker version